graphql/resolvers.go

```diff
@@ -2425,6 +2433,22 @@ func (r *taskResolver) Annotation(ctx context.Context, obj *restModel.APITask) (
 	return apiAnnotation, nil
 }
 
+func (r *annotationResolver) Issues(ctx context.Context, obj *restModel.APITaskAnnotation) ([]*restModel.DisplayTicket, error) {
+	displayTickets, err := restModel.GetJiraTickets(obj.Issues)
```

Comment:
I don't think we need Issue / SuspectedIssues queries. We're always going to query the whole annotation at one time, so we should get display ticket information in the Annotation resolver.